dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/Parent.java

```diff
@@ -142,7 +142,7 @@ protected void loadDockerfileGithubUtil(DockerfileGitHubUtil _dockerfileGitHubUt
             }
         }
 
-        log.info("Path to Dockerfiles in repo '{}': {}", parentRepoName, pathToDockerfilesInParentRepo);
+        log.info("Path to Dockerfiles in repos: {}", pathToDockerfilesInParentRepo);
```

Comment:
`parentRepoName` was the last repo (which doesn't really make sense when you're printing something for all of the repos.